pwnlib/tubes/server.py

```diff
@@ -0,0 +1,157 @@
+from __future__ import absolute_import
+
+import errno
+import socket
+
+from pwnlib.context import context
+from pwnlib.log import getLogger
+from pwnlib.tubes.sock import sock
+from pwnlib.tubes.remote import remote
+
+log = getLogger(__name__)
+
+class server(sock):
+    r"""Creates an TCP or UDP-server to listen for connections. It supports
+    both IPv4 and IPv6.
+
+    The callback function should take a :class:`pwnlib.tubes.remote` as
```

Comment:
This bit of documentation should go on the `callback` argument documentation below